includes/class-wcpdf-admin.php

```diff
@@ -307,16 +313,18 @@ public function request_query_sort_by_column( $query_vars ) {
 	}
 
 	/**
-	 * Add the meta box on the single order page
+	 * Add the meta boxes on the single order page
 	 */
 	public function add_meta_boxes() {
+		$screen = wc_get_container()->get( CustomOrdersTableController::class )->custom_orders_table_usage_is_enabled() ? wc_get_page_screen_id( 'shop-order' ) : 'shop_order';
```

Comment:
@alexmigf Won't there be a fatal error for the non-existent class `CustomOrdersTableController` before WC 7.1?